source/objects/base.lua

```diff
@@ -13,7 +13,7 @@ local Base = {}
 -- Local Variables
 -- ~~~~~~~~~~~~~~~~
 
-local landingLights     = Assets.getImageSet("landingLights")
+local landingLights     = Assets.getImageSet("landingLights2")
 landingLights.animation = Assets.newAnimation("landingLights", landingLights.image, 64, 8, 1, '1-4', 0.5)
```

Comment:
Why renaming the file to landingLights2 instead of landingLights? Is the old file still used? Asking out of curiosity.